### PR TITLE
style: basic layout styling for feedback survey

### PR DESCRIPTION
**Pull request by amy-corson-ibigroup**
- Basic layout styling for all survey question types
- Adds optional "subtitle" field for `info`  questions

### CODE REVIEW

File: components/RadioButtons.tsx
Comment (AdrianaCeric) [inline components/RadioButtons.tsx]: Just as heads up for a11y heading hierarchy, my PR will [change the `h3` to an `h2`](https://github.com/ibi-group/survey/pull/17/files#diff-5bc2b5087dcba0ccdfe1abf46b3613772b9d3b43eba6b8a748bc3eda588cd3c3R42-R51). 
The css h3 selectors should also be changed to h2.

**Review by AdrianaCeric — APPROVED** [review]
Changes look great! Love the minimalistic black and white look. Just realized our two PRs are working on the same file components. There are structural changes (for a11y) that I made to each title like adding a `<legend>` to each `<fieldset>` and switching h3 --> h2. This PR may be blocked until that one's merged?

File: components/QuestionRenderer.tsx
Comment (miles-grant-ibigroup) [inline components/QuestionRenderer.tsx]: Can we make this an h2 instead? Then we don't need a class on it either

File: components/RadioButtons.tsx
Comment (miles-grant-ibigroup) [inline components/RadioButtons.tsx]: The other questions now have centered question titles, but this one isn't centered. Do we want to move those centering styles to the global h2/h3 style instead?

File: components/Stars.tsx
Comment (miles-grant-ibigroup) [inline components/Stars.tsx]: Clever, but why is the star container different?

File: config.yaml
Comment (miles-grant-ibigroup) [inline config.yaml]: Can we rename this `completionSubtitle` for consistency? We'll have to make it a breaking change

File: config.yaml
Comment (miles-grant-ibigroup) [inline config.yaml:57]: Lol I like this

File: pages/index.tsx
Comment (miles-grant-ibigroup) [inline pages/index.tsx]: Why did we change this to a span?

File: styles/RadioButtons.module.css
Comment (miles-grant-ibigroup) [inline styles/RadioButtons.module.css]: image
Is this intentional? I think it's a bit funky

File: styles/RadioButtons.module.css
Comment (miles-grant-ibigroup) [inline styles/RadioButtons.module.css]: I like the new paddings, but I'd like to try to stick to `.5` increments if possible just to keep grids clean

File: styles/globals.css
Comment (miles-grant-ibigroup) [inline styles/globals.css:10]: I agree it looks cleaner with white but could we make this configurable?

File: styles/globals.css
Comment (miles-grant-ibigroup) [inline styles/globals.css]: Should we have all font sizes in `em`?

File: styles/globals.css
Comment (miles-grant-ibigroup) [inline styles/globals.css]: Why are we doing this instead of flex box? Or if you want to get really fancy putting a `display: grid` on the container

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
I really like the end product I think it looks clean and I like how the new layout looks. Just some nitpicks on the implementation!

Also I think we should consider gridding the radio buttons/entire container -- I don't like the current reflow setup. I think they should all be in 1 row and if that doesn't fit one per row. This is easy to accomplish with `minmax` and grid

File: components/RadioButtons.tsx
Comment (AdrianaCeric) [reply components/RadioButtons.tsx]: That sounds like a good idea

File: components/Stars.tsx
Comment (amy-corson-ibigroup) [reply components/Stars.tsx]: It was because they used to share `display` styles! No longer relevant though.

File: pages/index.tsx
Comment (amy-corson-ibigroup) [reply pages/index.tsx]: To match the first subtitle! I've changed both to `h2`s

File: styles/RadioButtons.module.css
Comment (amy-corson-ibigroup) [reply styles/RadioButtons.module.css]: I thought this was a little more of a standard focus indicator than the dashed outline. Open to alternative ideas!

Comment (miles-grant-ibigroup) [reply styles/RadioButtons.module.css]: I think coloring the outline is more standard than adding a second layer

File: components/Alert.tsx
Comment (miles-grant-ibigroup) [inline components/Alert.tsx]: I think this was intentionally not done to improve the animation

File: pages/index.tsx
Comment (miles-grant-ibigroup) [inline pages/index.tsx]: This needs to be hidden in the case that the key doesn't exist!

**Review by miles-grant-ibigroup — CHANGES_REQUESTED** [review]
Looking really good! Tried to find a way to get the grid looking right, but to no success. Just a few nitpicks, and also the :focus border. But looking great overall!

File: styles/RadioButtons.module.css
Comment (miles-grant-ibigroup) [inline styles/RadioButtons.module.css:4]: This is still not quite working right for me, but I think it's fine for now!

**Review by miles-grant-ibigroup — APPROVED** [review]
Looking really clean! Thanks for all your work on this

**Pull request merged**
